pkg/build/build.go

```diff
@@ -266,9 +268,17 @@ func (b *Build) Run(ctx context.Context, recreateCluster bool) error {
 		return fmt.Errorf("creating localbuild resource: %w", err)
 	}
 
-	err = <-managerExit
-	close(managerExit)
-	return err
+	interrupted := make(chan os.Signal, 1)
```

Comment:
Is there a way we could move this "out" since its really a global cancellation, not simply cancelling this subroutine. I'm used to this getting set up along with other intialization, usually in the main() method. We should be able to cancel a context there which cascades down in to cancelling this context.

Reply:
Ah good idea. Will move it up.

Reply:
@greghaynes done.